apps/website/.vuepress/config.js

```diff
@@ -125,8 +125,7 @@ module.exports = {
         '@clr/core/textarea': path.resolve('../../dist/core/textarea'),
         '@clr/core/time': path.resolve('../../dist/core/time'),
         '@clr/core/toggle': path.resolve('../../dist/core/toggle'),
-        '@clr/icons': path.resolve('../../dist/clr-icons'),
-        '@clr/icons/shapes/all-shapes': path.resolve('../../dist/clr-icons/shapes/all-shapes.min'),
+        // @TODO Would like to be able to remove these
```

Comment:
@gnomeontherun Are you sure these are needed? Previously we needed to build the `core` package in the website build process and that's why we needed these packages but after we resolved the build process I think these packages are no longer needed here.